templates/CRM/common/civicrm.settings.php.template

```diff
@@ -495,6 +495,11 @@ if (!defined('CIVICRM_CLEANURL')) {
   }
 }
 
+// @deprecated This constant is deprecated and will be removed.
```

Comment:
I think you should drop these lines out -

Reply:
Ok.

Also undeprecated the methods, but I do think they provide an unnecessary feature.  I can't think of a scenario where we want to use the database server's default character set and collation, whatever that might be.

Reply:
yeah - I'm half-way convinced there - mark this as not draft & let's let jenkins have at it